### PR TITLE
Called Gripper gloves - have nothing to do with gripping

### PR DESCRIPTION
**Pull request by optimumtact**
## About The Pull Request
Renames the gripping gloves to Enhanced Retrieval gloves

## Why It's Good For The Game
These gloves in fact do nothing for your grip, they actually only let you tackle things, alternate name suggestions are welcome.

## Changelog

:cl: oranges
spellcheck: Gripper gloves are now Enhanced Retrieval gloves
/:cl:

### CODE REVIEW

**Review by Jacquerel — APPROVED** [review]
I am afraid we have been instructed to crack down on people who do not adequately fill in "why their change is good for the game", the PR template is not optional and we ask contributors to fill it out properly so that we can properly understand what the reasoning behind a change is

**Comment by the-orange-cow** [comment]
Tiger Glove(note the gloves have nothing to do with tigers)

**Comment by CRITAWAKETS** [comment]
gorilla gloves don't make you a gorilla... though they could just be made out of gorilla fur
gorilla warfare ain't tackling bud

**Comment by DrAmazing343** [comment]
Grapple Gloves, because grappling is what you do when you hit a really good tackle and get the aggressive grab on them, and it rhymes with tackle. If people think it's like grappling hooks, it's a reading comprehension skill issue :3

**Comment by SomeoneYouProbablyKnow** [comment]
I hardly know er

File: code/modules/clothing/gloves/tacklers.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /obj/item/clothing/gloves/tackler
-	name = "gripper gloves"
-	desc = "Special gloves that manipulate the blood vessels in the wearer's hands, granting them the ability to launch headfirst into walls."
+	name = "Enhanced Retrieval gloves"
```
Comment (Fikou) [inline code/modules/clothing/gloves/tacklers.dm:2]:
```suggestion
	name = "enhanced retrieval gloves"
```
we don't capitalize names unless it's some brand it looks stupid

Comment (carlarctg) [reply code/modules/clothing/gloves/tacklers.dm:2]: go easy on contributors man wtf :/

**Comment by ju45he** [comment]
Can't wait for the E-sword to get renamed to "clown's baton" because it is more related to what it is actually used for.

**Comment by deathrobotpunch** [comment]
can’t wait for the supermatter to be called the electric rock

**Pull request merged**